k8s/kube-base/deployment-xdmod.yaml

```diff
@@ -1,20 +1,22 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: moc-xdmod
+  name: xdmod-ui
+  labels:
+    component: xdmod-ui
 spec:
   selector:
     matchLabels:
-      app: xdmod
+      component: xdmod-ui
```

Comment:
Sorry, I am a little confused. You deleted `k8s/overlays/nerc-ocp-infra/patches/deployment-xdmod.yaml`, which would add the label `app: xdmod` to the deployment named `moc-xdmod`. But now that the deployment name is changed to xdmod-ui and there's no patch in the overlay, how will the app label be added here?  Or do you just not plan to add it.

Reply:
Also, that patch was setting some container images too, is that not needed anymore?

Reply:
Not quite how it seems to work.

app: xdmod is being added by the k8s/kube-base/kustomization file though the commonLabels section:

        commonLabels:
          app: xdmod

I decided to change the name of the deployment to match the component as it is more descriptive of what it is deploying.  That is the xdmod-ui.  moc-xdmod doesn't really seem to fit any more, as if it ever did, and xdmod is built of several different pieces so I am trying to more descriptive of what each piece is.

k8s/overlays/nerc-ocp-infra/patches/deployment-xdmod.yaml was not being referred to in the k8s/overlays/nerc-ocp-infra/kustomization fils:

        kind: Kustomization
        resources:
          - ../../xdmod-build
          - ../../kube-base
        patchesStrategicMerge:
          - patches/bc-xdmod.yaml
          - patches/bc-xdmod-dev.yaml

So why keep it?

Reply:
@rob-baron thank you, that makes sense.